src/test/resources/mocks/binlookup/get3dsavailability-success.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Same as above, we can add some mock responses for invalid scenarios like invalid data etc.

Reply:
Added